src/component/getThought/GetThought.jsx

```diff
@@ -0,0 +1,121 @@
+import {useState, useEffect} from "react";
+import { NewThoughtsForm } from "../newThought/NewThoughtsForm";
+import moment from "moment";
+import "./getThought.css";
+
+export const GetThought = () => {
+    const [oldThoughts, setOldThoughts] = useState([])
+    const [loading, setLoading] = useState(true)
+    const [newThoughts, setNewThoughts] = useState('')
+    const [error, setError] = useState(null)
+
+    const URL = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts'
+
+    const handleInputChange = e => setNewThoughts(e.target.value)
+
+    useEffect (()=>{
+      setLoading(true)
+        const fetchData = async () =>{
+          try {
+            const res = await fetch(URL)
+            const data = await res.json()
+            setOldThoughts(data)
+          } catch (error) {
+            console.error('Error fetching data:', error)
+          } finally {
+            setLoading(false)
+          }
+        }
+        fetchData()
+  
+        const intervalID = setInterval(fetchData, 5000)
+  
+        return () => {
+          clearInterval(intervalID)
+        }
+      }, [])
+
+      const handleFormSubmit = (e) => {
+        e.preventDefault()
+
+        if (newThoughts.length === 0){
+          setError(alert('You can not send an empty message.'))
+        } else if (newThoughts.length < 5 || newThoughts.length >140){
+          setError(alert('Please write 5 to 140 characters.'))
+        } else ('Typing undefine')
+
+        fetch(URL,{
+            method: 'POST',
+            body: JSON.stringify ({message: newThoughts}),
+            headers: { "Content-Type": "application/json" },
+        })
+          .then(res => res.json())
+          .then((newThought) => {
+            setNewThoughts('')
+            setNewThoughts((oldThoughts)=> [newThought, ...oldThoughts])
+            setError(null)
+          })
```

Comment:
New thought are not updated until reloading the page.

Reply:
I've changed the line 55 to `setNewThoughts(newThought)` now it works and the [object] disappeared